fixes #3654 (#3655)
knative/docs: docs/getting-started/first-service.md

# Deploying your first Knative Service
!!! tip
    Hit ++"n"++ / ++"."++ on your keyboard to move forward in the tutorial. Use ++"p"++ / ++","++ to go back at any time.

> ==**In this tutorial, we are going to deploy a "Hello world" Service!**==

Reviewer: ```suggestion
==**In this tutorial, you can deploy a "Hello world" service.**==
```

    spec:
      template:
        metadata:

> # This is the name of our new "Revision," see "?" box below for more details

Reviewer 2: Seems this change makes this look awkward on the website

Reviewer 3: Just noticed, should have used the netlist preview. 

Maybe the comment needs to be indented

Author: @csantanapr opened up #3661 for this
